### ISSUE TITLE
Worker scout is preventing us from expanding

### ISSUE THREAD

**Issue by RaphaelRoyerRivard** [RaphaelRoyerRivard/MicroMachine#811]
We should allow the creation of production buildings before the expansion if we have too many resources.
![image](https://user-images.githubusercontent.com/5833235/77827000-84725e00-70e9-11ea-9ab2-d05e07dc99f5.png)

**Comment by Unusual229** [comment]
You mean the scouting worker is generating influence that is blocking our expand?

**Comment by RaphaelRoyerRivard** [comment]
Yeah we aren't sending a worker to the natural because the enemy worker generates influence and we don't send any unit to kill it because it stays never long enough in our base.

**Comment by Unusual229** [comment]
Any way I can know the influence is generated by a worker?

**Comment by RaphaelRoyerRivard** [comment]
Not currently...
